packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2775,29 +2777,56 @@ export const audiusBackend = ({
     }
   }
 
+  /**
+   * Returns whether the current user is subscribed to userId.
+   * @returns Boolean
+   */
   async function getUserSubscribed(userId: ID) {
     await waitForLibsInit()
     const account = audiusLibs.Account.getCurrentUser()
     if (!account) return
-    try {
-      const { data, signature } = await signData()
-      return await fetch(
-        `${identityServiceUrl}/notifications/subscription?userId=${userId}`,
-        {
-          headers: {
-            [AuthHeaders.Message]: data,
-            [AuthHeaders.Signature]: signature
+
+    const readSubscribersFromDiscoveryEnabled =
```

Comment:
nit, but we try to prefix booleans with `is`

Reply:
added in 37d3ffce99d3ea6ae2137e569042e69810a15b64